### PR TITLE
Add empymod

### PR DESCRIPTION
**Pull request by prisae**
Checklist

- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vendor other packages.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by prisae** [comment]
Running `conda smithy` yields this, no idea what the issue is...?
```
(condaforge) empymod/> conda smithy recipe-lint .
Traceback (most recent call last):
  File "/home/dtr/anaconda3/envs/condaforge/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "/home/dtr/anaconda3/envs/condaforge/lib/python3.7/site-packages/conda_smithy/cli.py", line 485, in main
    args.subcommand_func(args)
  File "/home/dtr/anaconda3/envs/condaforge/lib/python3.7/site-packages/conda_smithy/cli.py", line 380, in __call__
    return_hints=True,
  File "/home/dtr/anaconda3/envs/condaforge/lib/python3.7/site-packages/conda_smithy/lint_recipe.py", line 597, in main
    results, hints = lintify(meta, recipe_dir, conda_forge)
  File "/home/dtr/anaconda3/envs/condaforge/lib/python3.7/site-packages/conda_smithy/lint_recipe.py", line 397, in lintify
    req, _, _ = requirement.partition("#")
AttributeError: 'CommentedMap' object has no attribute 'partition'
```

**Comment by prisae** [comment]
Hi @conda-forge/help-python, this PR is ready for review. Thanks a lot!

File: recipes/empymod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dba7b900b2fe0e811d8cae47cb53d81a05e851e1e9b21cecc37bb3d77da42cf5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  preserve_egg_dir: True
```
Comment (beckermr) [inline recipes/empymod/meta.yaml]: Just commenting to make sure you actually need this.

**Comment by prisae** [comment]
I needed that earlier on to get the version from `setuptools_scm`. If conda-forge can get the version without that, then I am happy to take it away!

File: recipes/empymod/meta.yaml (hunk 1 above)
Comment (prisae) [reply recipes/empymod/meta.yaml]: If I leave it out, does conda-forge get the right version with `setuptools_scm`?

**Comment by beckermr** [comment]
you know best! merging!

**Pull request merged**